src/Bramus/Router/Router.php

```diff
@@ -47,6 +47,20 @@ class Router
      */
     private $namespace = '';
 
+    /**
+     * @var static Parent Router ...
+     */
+    private $parent = '';
+
+    public function __construct($args = [])
```

Comment:
Missing document blocks for `__construct`. And please use `array()` instead of `[]`